apps/bfd-pipeline/bfd-pipeline-rda-bridge/run_bridge.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
 PROJ_DIR=$(dirname "$0")
+VERSION_NUM=$2
```

Comment:
Turns out if the user doesn't provide a project version for $1, this won't work since it'll grab the output directory. I just used $(yq .project.parent.version "${PROJ_DIR}/pom.xml") instead!